platform/mellanox/mlnx-platform-api/sonic_platform/sfp.py

```diff
@@ -283,59 +282,41 @@
 # Global logger class instance
 logger = Logger()
 
+
+# SDK initializing stuff, called from chassis
+def initialize_sdk_handle():
+    rc, sdk_handle = sx_api_open(None)
+    if (rc != SX_STATUS_SUCCESS):
+        logger.log_warning("Failed to open api handle, please check whether SDK is running.")
+        sdk_handle = None
+
+    return sdk_handle
+
+def deinitialize_sdk_handle(sdk_handle):
+    if sdk_handle is not None:
+        rc = sx_api_close(sdk_handle)
+        if (rc != SX_STATUS_SUCCESS):
+            logger.log_warning("Failed to close api handle.")
+
+        return rc == SXD_STATUS_SUCCESS
+    else:
+         logger.log_warning("Sdk handle is none")
+         return False
```

Comment:
Since these functions are not called from within this module and are only called from chassis.py, why define them here instead of in chassis.py? It seems like chassis.py is responsible for managing the SDK handle and passing it around so these functions feel like they belong there.

Reply:
@jleveque  after did the change found that it will break the uni-test, to move the SFP init/deinit functions to chassis then we need to import some SDK specific lib, in the uni-test environment, these libs are not available, so uni-test will be broken. Currently, we don't have a good solution to resolve the import issue for uni-test cases,  so would like to keep it as is, what's your view?

Reply:
We can keep it as-is for now, then. This can be saved for a future refactor.